### PR TITLE
Database setup/MongoDB model

### PR DESCRIPTION
**Pull request by nightlaro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
+    "@babel/preset-env": "^7.10.2",
+    "@babel/preset-react": "^7.10.1",
+    "babel-loader": "^8.1.0",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
+    "express": "^4.17.1",
+    "html-webpack-plugin": "^4.3.0",
+    "mongoose": "^5.9.19",
+    "prettier": "2.0.5",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```
Comment (RaneWallin) [inline package.json:27]: react, react-dom, express, and mongoose should all be dependencies, not devDependencies

Comment (nightlaro) [reply package.json:27]: Thank you! Will fix that :)

**Comment by nightlaro** [comment]
> * I set up mongo with a slightly different pattern, but I believe the schema and model go inside the callback for db.once.  Probable good to at least check on that.
>   -For separation and also for testing, recommend putting App component into compontents folder with the other components you might have.  Import App.jsx into index.js and render it with ReactDOM. That would be the only task for index.js.
>   -recommend install nodemon and use in start script in package.json
>   -Definitely delete bundle, and add it to your gitignore file before building again.  also add .env file to gitignore.  Save gitignore, then create .env file to store sensitive data and port #s etc.  will need to use something like npm dotenv installed to put those into process.env

Will look into these too, thank you for that. as for MongoDB, will test out putting schema/model in db.once, I was under the impression that db.once was only for connection testing and creating a new model automatically opens a connection. I haven't tested any of these so I could totally be wrong but will also test your suggestion! Thank you!

**Comment by Chris-Hesterman** [comment]
I am thinking I was wrong about that db.once callback.  I say if your way is working then it's correct.  the docs are somewhat misleading?
image
I'm going to set up my mongo that way to check it out.  Been going with treating connection as a promise.

**Pull request closed without merging**